apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -140,25 +139,25 @@ impl Default for MetricsCommon {
 
 /// Tracing configuration
 #[derive(Clone, Default, Debug, Deserialize, JsonSchema)]
-#[serde(deny_unknown_fields, rename_all = "snake_case")]
+#[serde(deny_unknown_fields, rename_all = "snake_case", default)]
```

Comment:
NIT: Could we get rid of the `rename_all` here as well?

Reply:
[99476b7](https://github.com/apollographql/router/pull/3968/commits/99476b7ac34f96453ce2ac75384a867574afb0b6)